src/server/database/Database/MySQLConnection.cpp

```diff
@@ -218,7 +218,9 @@ bool MySQLConnection::Execute(PreparedStatementBase* stmt)
     uint32 index = stmt->GetIndex();
 
     MySQLPreparedStatement* m_mStmt = GetPreparedStatement(index);
-    ASSERT(m_mStmt); // Can only be null if preparation failed, server side error or bad query
+    //ASSERT(m_mStmt); // Can only be null if preparation failed, server side error or bad query
+    if (!m_mStmt)
+        return false;
```

Comment:
```suggestion
    if (!m_mStmt)
    {
        return false;
    }
```

Reply:
@FrancescoBorzi you should check our latest code style convention. The first one was correct.